rust/template/src/lib.rs

```diff
@@ -79,11 +79,12 @@ pub fn string_append(mut s1: String, s2: &String) -> String {
     s1
 }
 
-/// A default implementation of `ConvertRelId` that just forwards calls
+/// A default implementation of `DDlogConvert` that just forwards calls
 /// to generated functions of equal name.
-pub struct RelIdConverter {}
+#[derive(Debug)]
+pub struct DDlogConverter {}
 
-impl ConvertRelId for RelIdConverter {
+impl DDlogConvert for DDlogConverter {
```

Comment:
`record.rs` already defines `FromRecord` and `IntoRecord` traits that are similar in spirit in that they convert between program-specific and generic types. I feel that going forward this trait should become part of the same framework.

Reply:
Sure!